nginx/cernopendata.conf

```diff
@@ -22,11 +22,16 @@ map $upstream_http_content_length $nocache {
 }
 
 server {
-    listen 80 default;
+    listen 80;
+    listen 443 ssl; # remove in production, SSL termination managed by OpenShift
     server_name _ "";
     server_tokens off;
     charset utf-8;
 
+    # remove in production, SSL termination managed by OpenShift
```

Comment:
This may get unnoticed as the time goes by... Two possibilities to improve the visibility:

- e.g. starts with `# FIXME remove in production...` since we usually grep for FIXME, so this won't get unnoticed.
- e.g. rename file to `ngix/localhost.conf` to indicate this is for local developments only.

Reply:
If we go for `localhost.conf` in `Dockerfile`, can we also rename `cernopenadat.crt` and `cernopendata.key` to `localhost.crt` and `localhost.key`, so that they would be consistent?

We could actually even remove `cernopendata.conf` if you keep those two `FIXME-remove-in-production` comment lines to decorate the snippets... So that we can test locally the exact file that will be deployed in production.

```diff
25c25,26
<     listen 80 default;
---
>     listen 80;
>     listen 443 ssl;
28a30,32
> 
>     ssl_certificate     /etc/ssl/cernopendata.crt;
>     ssl_certificate_key /etc/ssl/cernopendata.key;
```